runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -155,13 +161,20 @@ public VersionLockedObject(Supplier<T> newObjectFn,
      * gc and external object access is needed.
      */
     public void gc(long trimMark) {
+        Timer vloGcTimer = VloMetricsParams.metrics.timer(VloMetricsParams.VLO_GC);
+        Timer.Context vloGcDuration = MetricsUtils.getConditionalContext(vloGcTimer);
+
         long ts = lock.writeLock();
+
         try {
             pendingUpcalls.removeIf(e -> e < trimMark);
             upcallResults.entrySet().removeIf(e -> e.getKey() < trimMark);
             smrStream.gc(trimMark);
         } finally {
             lock.unlock(ts);
+            if (vloGcDuration != null) {
```

Comment:
is it possible `vloGcDuration != null` ?

Reply:
yes